GeneralsMD/Code/Libraries/Source/WWVegas/WWDownload/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+# Set source files
+set(WWDOWNLOAD_SRC
+    Download.cpp
+    Ftp.cpp
+    registry.cpp
+    urlBuilder.cpp
+    Download.h
+    DownloadDebug.h
+    DownloadDefs.h
+    Ftp.h
+    FtpDefs.h
+    Registry.h
+    urlBuilder.h
+)
```

Can you please match the case of the files.
```suggestion
set(WWDOWNLOAD_SRC
    Download.cpp
    FTP.CPP
    registry.cpp
    urlBuilder.cpp
    Download.h
    DownloadDebug.h
    downloaddefs.h
    ftp.h
    ftpdefs.h
    Registry.h
    urlBuilder.h
)
```

---

nice catch. Thank you.